REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.3
 MathProgBase 0.3.8
 ECOS 0.4.1
+FactCheck 0.2.5
```

Comment:
What would be the nicest way for FactCheck to still be present for travis?
JuMP seems to be doing this: https://github.com/JuliaOpt/JuMP.jl/blob/master/.travis.yml#L11